### PR TITLE
Event item

### LINKED ISSUE #102: Calendar item
- [ ] Implement the most basic calendar item (see the one below as an example - do not implement functionality to expand the calendar item yet or any tags like "Beginner friendly")
Screenshot 2022-12-15 at 12 12 17 AM

- [ ] Make this responsive on mobile + desktop 
- [x] When you have a bunch of these schedules, make sure you can scroll through them in the green thingy 
Screenshot 2022-12-15 at 12 13 54 AM

### PR DESCRIPTION
**Pull request by srazh**
<Summary line - One sentence describing your intended set of changes>

Fixes #102 .

Changelist:

- implemented the most basic calendar event component 
- used + mapped dummy data to display scrollable events
- responsive for all breakpoints 

Notes:

- got rid of outline for event because I thought it looked too busy 

Tested:

- local host 8000

Screenshots & Screencasts:

Uploading Screen Recording 2022-12-19 at 8.45.58 PM.mov…

709D3144-1D3A-471D-B30F-4CA0FDD79245
C8C08F70-0DF4-4A5D-A4EE-798848F0C394
D46138F9-FEB1-4895-BB8E-CE0E6AD4BCDD
0D4F9744-48F8-409A-97C2-76BBBF016EAE

### CODE REVIEW

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: Use em!

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx:188]: Run formatter on all files to format

File: live-site/components/event-schedule-section/event-item/EventItem.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.tsx]: remove console log

File: live-site/components/event-schedule-section/event-item/EventItem.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.tsx]: Remove all of the extra lines in the file

File: live-site/components/event-schedule-section/EventScheduleTab.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleTab.tsx:15]: For better code organization lets move this file and the style file into a new folder /components/event-schedule-section/event-schedule-tab

File: live-site/components/event-schedule-section/event-item/EventItem.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.tsx]: OK i figured out why we have the extra eventItem

So online 21 we need to wrap eventItem in curly brackets like ({eventItem}) - we need to do this whenever we pass in props I think 

and then u can change everything else to be eventItem.time eventItem.eventType, etc.

File: live-site/lib/data.ts
Comment (jzhzhang66) [inline live-site/lib/data.ts:45]: Remove all of this extra space / lines

File: live-site/lib/types.ts
Comment (jzhzhang66) [inline live-site/lib/types.ts]: remove all of this extra space / lines

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
THIS IS LOOOKING GREAT SARAH!!!! 

Thanks so much for working on this~ 

A few comments in addition to the ones I left below: 
- There is a mobile design so on mobile could you please change the implementation to reflect that? Right now mobile just looks like desktop but more squished 
Screenshot 2022-12-20 at 2 41 03 PM
- The title text like "Opening ceremony" looks larger than on the Figma. On the Figma its roughly the same size as the time. Could we please make that smaller?
Screenshot 2022-12-20 at 2 41 48 PM
- At certain breakpoints, the arrows are in the wrong spot so just make sure they're in the right spot at every screensize 
Screenshot 2022-12-20 at 2 42 51 PM
- I think the green calendar container thing can actually be longer - it seems longer on the Figma since it can contain 5 items
Screenshot 2022-12-20 at 2 43 41 PM

File: live-site/lib/data.ts
Comment (mike10911) [inline live-site/lib/data.ts]: All the eventTypes should be capitals to reflect design!

**Review by mike10911 — CHANGES_REQUESTED** [review]
This looks stunning!!! +1 to all of Judy's comments! I just have 3 minor changes.

Screen Shot 2022-12-20 at 11 14 55 PM

- Here is the design. Figma has a bit more padding between the green content square and the event items so we will want to 
reflect that.

- Also the thin blue border around each event item needs to be implemented

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: I think you can remove this padding-left and it should work because padding-left: 1em is also under max.tabletLg

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx:48]: Same here

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: I might be trippin but do these two styles do anything? I removed them and it looks the same LOL

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx:159]: Screenshot 2022-12-31 at 11 53 43 AM

I'm still seeing this kind of issue - I think a good way to fix this is to make the StyledArrow position absolute if it's hard to get the arrow to fit. 

You'll also need to make StyledEventItemContainer position: relative since we want this element to be absolute relative to that container.

I'd recommend removing all the StyledArrow code and starting with something like this

`const StyledArrow = styled.img`
  position: absolute;
  right: 1em;
  top: 3em;
`;`

and then fixing the right, top, and width values as necessary.

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx:141]: Remove this padding-left and the other one on the StyledEventItemHeader

Because we want them to be aligned, we can just apply any padding-left we want to just the StyledEventItemNameLocationContainer instead of doing it twice

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
This looks a bit too close to the edge - can we add a bit more padding-left here 
Screenshot 2022-12-31 at 12 00 08 PM

It only switches to this design at super small screensizes - can we change this so the mobile design is implemented up until the tablet breakpoint? 
Screenshot 2022-12-31 at 12 01 41 PM

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Also, the blue border that Mike mentioned still needs to be implemented

THANKS SARAH FOR UR HARD WORK<33333

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: can we make margin-right: 1em; so that the items are centered? Right now they're closer to the left because of the different margins
Screenshot 2023-01-03 at 11 13 50 AM

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove random newlines

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove newlines

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-left: 1em; b/c we want these two to be aligned

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-left

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-left

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: padding-left: 0; 

no need for the em if it's 0

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: padding-top: 0 
no need for em if its 0

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: this is repetitive since you already have the font-size: 1em on line 112

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove this since you already have this on line 180

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove this entire thing (lines 193 - 195) since we already have this same style for max.tablet

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-left here since its repetitive from line 151

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove this since its repetitive as line 30

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove since its repetitive as line 21

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove since its repetitive as line 51

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Thank you for making those changes this is looking great!!!

I just went through and caught a bunch of small things that we can improve (like repeated styles, etc.) but I'll be ready to approve after 

Thank you for your hard work!!!

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-top here b/c duplicate

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-left

File: live-site/components/event-schedule-section/event-item/EventItem.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/event-item/EventItem.styles.tsx]: remove padding-right

**Review by jzhzhang66 — APPROVED** [review]
APPROVING WITH 3 MINOR COMMENTS!!!! (just more duplicates so be sure to remove those before merging)

**Review by mike10911 — APPROVED** [review]
LGTM!!!!

**Pull request merged**
